tasmota/settings.h

```diff
@@ -121,7 +121,7 @@ typedef union {                            // Restricted by MISRA-C Rule 18.4 bu
     uint32_t zerocross_dimmer : 1;         // bit 17 (v8.3.1.4)  - SetOption99 - Enable zerocross dimmer on PWM DIMMER
     uint32_t remove_zbreceived : 1;        // bit 18 (v8.3.1.7)  - SetOption100 - Remove ZbReceived form JSON message
     uint32_t zb_index_ep : 1;              // bit 19 (v8.3.1.7)  - SetOption101 - Add the source endpoint as suffix to attributes, ex `Power3` instead of `Power` if sent from endpoint 3
-    uint32_t spare20 : 1;
+    uint32_t buzzer_freq_mode : 1;         // bit 20 (v8.3.1.8)  - SetOption102 - Use frequency output for buzzer pin instead of on/off signal
```

Comment:
You need to catch up with setoptions now.

Reply:
I've had to learn how to do that (merge with upstream) and moved the setting to SetOption110. I hope I didn't screw it up, if I did I am sorry. I am not completely fluent in git yet ...

Reply:
OK, I made a mess in the meantime while trying to fix it. Did this cause the pull request to close? It should be in a good state now, though, with only the two relevant files changed compared to current upstream development.

Do you advise me to delete this pull request and open a new one, or can you re-open this one?

Reply:
It is now using the free SetOption110, but I am unsure about the version note in the comment and the free_f*a[] change. Can you just change it as appropriate according to your conventions and merge?

Reply:
Caught up with setoptions and is now using SetOption111.